docker/puppetserver-standalone/docker-entrypoint.d/70-set-dns-alt-names.sh

```diff
@@ -4,11 +4,17 @@
 # setting will only have an effect when the container is started without
 # an existing certificate on the /etc/puppetlabs/puppet volume
 if test -n "${DNS_ALT_NAMES}"; then
+    config_section="master"
```

Comment:
Could we have just set this in `main` instead of branching the logic? I may need to brush up on my config file processing, but I think that should then be used by either `master` or `agent` I believe.

Reply:
Yes I originally wrote it to just always use `[main]` but wasn't sure if that would have undesired side effects. I can submit another PR if you'd prefer that instead.